pages/common/du.md

```diff
@@ -14,14 +14,30 @@
 
 `du -sh {{path/to/directory}}`
 
+- The (x) is used to exclude NFS or other file types within the partition that is being scanned:
+
+`du -xsh {{path/to/directory}}`
+
 - List the human-readable sizes of a directory and of all the files and directories within it:
 
 `du -ah {{path/to/directory}}`
 
+- List the top 10 largest files in megabytes, sorted by largest:
+
+`du -Sm /var/log | sort -rn | sed '{11,$D; =}' | sed 'N; s/\n/ /' | awk '{printf $1 ":" "\t" $2 "\t" $3 "\n"}'`
+
 - List the human-readable sizes of a directory and any subdirectories, up to N levels deep:
 
 `du -h --max-depth=N {{path/to/directory}}`
 
 - List the human-readable size of all .jpg files in subdirectories of the current directory, and show a cumulative total at the end:
 
 `du -ch */*.jpg`
+
+- Number of files search:
+
+``echo "Detailed Inode usage for: $(pwd)" ; for d in `find -maxdepth 1 -type d |cut -d\/ -f2 |grep -xv . |sort`; do c=$(find $d |wc -l) ; printf "$c\t\t- $d\n" ; done ; printf "Total: \t\t$(find $(pwd) | wc -l)\n"``
```

Comment:
It is only one command per example. So if you want to add 3 commands, you need text for all 3. That's causing the error in Travis, too.

Reply:
Ok, Yea I have to encapsulate `find -maxdepth 1 -type d |cut -d\/ -f2 |grep -xv . |sort` this in an executable bash string that is not using the back tic too.  Also I thought that we could have up to 8 examples per command?